galasa-ui/src/app/myprofile/page.tsx

```diff
@@ -53,22 +69,49 @@ export default function MyProfilePage() {
       {isLoading ?
         <Loading data-testid="loader" small={false} active={isLoading} />
         :
-        <div className={styles.userNameContainer}>
-          <h4>Currently logged in as:</h4>
-          <h4> &nbsp; {loginId}</h4>
+        <div>
+          <h3 className={styles.loginActivityTile}>User Details</h3>
+          <div className={styles.userNameContainer}>
+            <h4>Currently logged in as:</h4>
+            <h4> &nbsp; {userData.loginId}</h4>
+          </div>
+
+          <h3 className={styles.loginActivityTile}>Recent Login Activity</h3>
+
+          {
+            clients.map((client, index) => {
+              return (
+                <div key={index} className={styles.loginActivityContainer}>
+                  <h4 className={styles.clientName}>
+                    {
+                      client.clientName === "web-ui"
+                        ?
+                        "Last logged in to this web application (UTC):"
+                        :
+                        "Last logged in using a Galasa personal access token (UTC):"
+                    }
+                  </h4>
+
+                  {/* Extracting the date and time from response */}
+                  <h4>&nbsp; {client.lastLogin.substring(0, 10)} {client.lastLogin.substring(11, 16)}</h4>
```

Comment:
Maybe converting the lastLogin to a Date and using that here would be easier to read?